.github/workflows/publish-gradle.yml

```diff
@@ -0,0 +1,21 @@
+name: GitHub Packages with Gradle
+
+on: push
```

Comment:
Should this only be done on the main branch?

Maybe in general have a build-gradle.yml, if not on the main branch.
And have this script publish-gradle.yml for the main branch.